### DISCUSSION TITLE
Oops: accidentally deleted 2-3 discussions

### DISCUSSION THREAD

**Discussion by sbvegan** [ethereum-optimism/developers#492] [Announcements]
We mistakenly deleted a few discussions while trying to clean up a different repository, sorry about that! If you were affected, please repost your thread and we'll get to it asap.

**Comment by imtipi** [comment]
cc @shaokun11 please repost if your L2 chain data reset issue still exist,because I found your thread disappeared when I want to discuss this in team meeting
